crates/rune-modules/src/macros.rs

```diff
@@ -0,0 +1,65 @@
+//! `std::macros` module for the [Rune Language].
+//!
+//! [Rune Language]: https://rune-rs.github.io
+//!
+//! ## Usage
+//!
+//! Add the following to your `Cargo.toml`:
+//!
+//! ```toml
+//! rune-modules = {version = "0.7.0", features = ["macros"]}
+//! ```
+//!
+//! Install it into your context:
+//!
+//! ```rust
+//! # fn main() -> runestick::Result<()> {
+//! let mut context = runestick::Context::with_default_modules()?;
+//! context.install(&rune_modules::macros::module(true)?)?;
+//! # Ok(())
+//! # }
+//! ```
+//!
+//! Use it in Rune:
+//!
+//! ```rust,ignore
+//! fn main() {
+//!     println(`Hello from ${file!()}:${line!()});
+//! }
+//! ```
+
+use rune::{Parser, TokenStream};
+
+/// Construct the supplemental `std::macros` module.
+pub fn module(_unused: bool) -> Result<runestick::Module, runestick::ContextError> {
+    let mut builtins = runestick::Module::new(&["std", "macros", "builtin"]);
```

Comment:
Guess this module is as good as any for now. Not sure where to put these macros since they just live immediately in `std` in Rust!

Reply:
Yeah; It's a bit odd. They're actually (code organization wise) in std/core/macros, in a `mod builtin`. Since I'm adding them to the prelude I guess it's not too important where they live apart from requiring the correct rune-modules features?

Reply:
Ah, this is a result of the `macro_export` causing macros to appear in the root. So I think this path is correct w.r.t. code organization but we don't have the same kludgy export mechanics that Rust has.

Reply:
Ah, yeah. That seems fine to me then :)